### PR TITLE
Improve long press support for most devices.

### PR DESCRIPTION
**Pull request by pastaq**
- Adds monitoring of multiple devices in asyncio loop to catch left meta, which is emitted by some devices when the power button is held for >1s. Has the benefit of blocking multiple power key events from triggering if more than one power device is present. Tested to work while InputPlumber is running. HHD not tested but should be able to be made compatible if the power button management is pulled from it.

- Reorganized the code and added some python conventions for script safety as well.

### CODE REVIEW

**Review by aarron-lee — APPROVED** [review]
lgtm! there's some small typos in the print statements, but that's not a big deal

I'll merge it after doing some testing later tonight.

**Comment by pastaq** [comment]
I can fix those tonight before you merge

**Comment by aarron-lee** [comment]
so as a followup, just tested this and the long press still doesn't work. I'm assuming the long press functionality only works with input plumber

short press still works fine, this was tested on my LGO + bazzite with hhd's power button management disabled.

I'll probably still merge this PR since the long press never originally worked anyways, so it's not like there's any difference in outcome.

**Comment by pastaq** [comment]
> so as a followup, just tested this and the long press still doesn't work. I'm assuming the long press functionality only works with input plumber
> 
> short press still works fine, this was tested on my LGO + bazzite with hhd's power button management disabled.
> 
> I'll probably still merge this PR since the long press never originally worked anyways, so it's not like there's any difference in outcome.

It depends on the device in question and a bit how HHD handles grabbing/passing events. 
Some power buttons will send a LEFT_META event on the AT Translated Set 2 Keyboard when the power button is held. If HHD is grabbing the AT Keyboard and not passing the unused events along then it wont work. Also, not all power buttons do this.

**Pull request merged**
